src/main/java/org/inferred/freebuilder/processor/util/CompilationUnitBuilder.java

```diff
@@ -100,12 +99,19 @@ public String toString() {
       }
       unit.append("\n");
     }
+    unit.append(formatSource(source.toString()));
+    return unit.toString();
+  }
+
+  private static String formatSource(String source) {
     try {
-      unit.append(new Formatter().formatSource(source.toString()));
-    } catch (FormatterException e) {
+      return new Formatter().formatSource(source);
+    } catch (UnsupportedClassVersionError e) {
+      // Formatter requires Java 7+; do no formatting in Java 6.
+      return source;
+    } catch (Exception e) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://www.codacy.com/app/Chris-Purcell-39/FreeBuilder/file/2405741245/issues/source?bid=3146243&fileBranchId=3233669#l112)